python-libraries/nanover-omni/src/nanover/omni/ase_omm.py

```diff
@@ -34,14 +34,28 @@ class InitialState:
 
 
 class ASEOpenMMSimulation:
+    """
+    A wrapper for ASE OpenMM simulations so they can be run inside the OmniRunner.
+    """
+
     @classmethod
     def from_simulation(cls, simulation: Simulation, *, name: Optional[str] = None):
+        """
+        Construct this from an existing ASE OpenMM simulation.
+        :param simulation: An existing ASE OpenMM Simulation
+        :param name: An optional name for the simulation
+        """
         sim = cls(name)
         sim.simulation = simulation
         return sim
 
     @classmethod
     def from_xml_path(cls, path: PathLike[str], *, name: Optional[str] = None):
+        """
+        Construct this from an existing NanoVer OpenMM XML file at a given path.
+        :param path: Path of the NanoVer OpenMM XML file
+        :param name: An optional name for the simulation
```

Comment:
What is the default name for the simulation if none is given?

Reply:
For a file it defaults to the filename, for an existing simulation it's something like `Unnamed ASE OpenMM Simulation" or something like that--I have updated these to be a little more specific